Formula/docker-commander.rb

```diff
@@ -0,0 +1,24 @@
+class DockerCommander < Formula
+  desc "Execute commands in docker containers"
+  homepage "https://github.com/daylioti/docker-commander"
+  url "https://github.com/daylioti/docker-commander.git",
+   :tag      => "1.1.4"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GO111MODULE"] = "on"
+    src = buildpath/"src/github.com/daylioti/docker-commander"
+    src.install buildpath.children
+    src.cd do
+      system "go", "build", "-ldflags", "-X main.version=#{version}"
+      bin.install "docker-commander"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Hi @SMillerDev 
Thank you for review.

My application is a GUI program in terminal, it have no functionality without GUI, except display version.

I think it will there be enough to check parsing yml (check if GUI error exist) configuration and compare versions, that sets in ldflags.

Could you check my changes again?